app/src/main/java/cm/aptoide/pt/search/view/SearchResultFragment.java

```diff
@@ -645,11 +648,33 @@ private boolean hasSearchResults() {
   }
 
   private void setupTheme() {
+    if (viewModel != null
```

Comment:
Could these conditions in the if clause also be replaced by a more readable call to a private method?